Package/Captcha/Captcha.php

```diff
@@ -106,6 +106,14 @@ class Captcha
      * @see https://developers.google.com/recaptcha/docs/display
      */
     protected $size = 'normal';
+    
+    /**
+     * Forces the widget to render in a specific language. Auto-detects the user's language if unspecified.
+     * 
+     * @var string
+     * @see https://developers.google.com/recaptcha/docs/language
+     */
+    protected $languageCode = null;
```

Comment:
Please move @see above @var if possible. You don't need to set = null; can get rid of that.